sumologic/data_source_sumologic_cse_log_mapping_vendor_product.go

```diff
@@ -0,0 +1,53 @@
+package sumologic
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceCSELogMappingVendorAndProduct() *schema.Resource {
+
+	return &schema.Resource{
+		Read: dataSourceCSELogMappingVendorAndProductRead,
+		Schema: map[string]*schema.Schema{
+			"guid": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"product": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"vendor": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+		},
+	}
+
+}
+
+func dataSourceCSELogMappingVendorAndProductRead(d *schema.ResourceData, meta interface{}) error {
+	c := meta.(*Client)
+
+	id := d.Id()
+
+	vendorAndProduct, err := c.GetCSELogMappingVendorsAndProducts(d.Get("product").(string), d.Get("vendor").(string))
+
+	if err != nil {
+		return err
+	}
+
+	if vendorAndProduct == nil {
+		d.SetId("")
+		return fmt.Errorf("Vendor product not found, removing from state: %v - %v", id, err)
```

Comment:
`err` will be nil here. Remove it from the error message?

Reply:
done under https://github.com/SumoLogic/terraform-provider-sumologic/pull/284/commits/a44f995f197a0d27ca41d773aa10652ebeba4e48